crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -779,6 +821,34 @@ impl<R: Resource> Command for RemoveResource<R> {
     }
 }
 
+/// A type erased [`Command`] that can be added to [`Commands`]
+pub struct BoxedCommand {
+    command: Box<dyn Command>,
+    func: unsafe fn(value: *mut u8, world: &mut World),
+}
+
+pub trait BoxableCommand {
+    /// Box the command, helpful when storing mixed commands together.
+    fn box_command(self) -> BoxedCommand;
+}
+
+impl<T: Command> BoxableCommand for T {
+    fn box_command(self) -> BoxedCommand {
```

Comment:
This function is wildy not safe and needs to be marked unsafe - I can call it with literary any type for insta UB. Only the caller can ensure it's called with `T`s that makes sense

Reply:
Right, you're saying it's always safe to call with commands given bound on T, I misread